docs/endpoints/multi_user.rst

```diff
@@ -456,6 +463,57 @@ a custom ``globus-compute-endpoint`` wrapper:
 
 (The use of ``exec`` is not critical, but keeps the process tree tidy.)
 
+
+Configuring to Accept Multiple Python Versions
+==============================================
+
+By default, Globus Compute serializes task submissions via `dill`_ with a method that
+uses Python bytecode. That's to say it does *not* serialize the source code unless asked
+to, for both technical and historical reasons, but this can sometimes cause issues.
+Because the underlying representations that Python uses for bytecode are subject to
```

Comment:
- Formally, consider spelling out `it's` to `it is`
- _... can sometimes cause issues_ is vague.  Are you able to be more precise, or give an example?  The following text starts to get at this, but I think the "can cause issues" is not necessary.  I'll not use this forum for the strict prose editing; we can chat out-of-GH-band if necessary.

Reply:
Not seeing `it's`, do you mean `that's`? (and not to be pedantic, but I see at least one usage of `that's` by you in this document :P )

I removed "can sometimes cause issues" and changed one instance of "that's" to "that is". I kept the instance here because "that is to say" doesn't flow as well as "that's to say" in my opinion, but I can be convinced otherwise.